cli/src/argparse/modification.rs

```diff
@@ -122,21 +138,33 @@ impl Modification {
         u.modifications.push(usage::Modification {
             syntax: "+TAG",
             summary: "Tag task",
-            description: "
-                Add the given tag to the task.",
+            description: "Add the given tag to the task.",
         });
         u.modifications.push(usage::Modification {
             syntax: "-TAG",
             summary: "Un-tag task",
+            description: "Remove the given tag from the task.",
+        });
+        u.modifications.push(usage::Modification {
+            syntax: "status:{pending,completed,deleted}",
+            summary: "Set the task's status",
+            description: "Set the status of the task explicitly.",
+        });
+        u.modifications.push(usage::Modification {
+            syntax: "wait:<timestamp>",
+            summary: "Set or unset the task's wait time",
             description: "
-                Remove the given tag from the task.",
+                Set the time before which the task is not actionable and
+                should not be shown in reports.  With `wait:`, the time
+                is un-set.  See the documentation for the timestamp syntax.",
```

Comment:
Maybe worth including a very short example like `wait:1day` as well as `wait:` to give some indication of the syntax?